gnovm/pkg/gnolang/store.go

```diff
@@ -900,13 +921,20 @@ func (ds *defaultStore) consumeGas(gas int64, descriptor string) {
 	}
 }
 
+// It resturns storage changes per realm within message
+func (ds *defaultStore) RealmDiffs() map[string]int64 {
+	return ds.realmDiffs
+}
+
 // Unstable.
 // This function is used to clear the object cache every transaction.
 // It also sets a new allocator.
 func (ds *defaultStore) ClearObjectCache() {
 	ds.alloc.Reset()
 	ds.cacheObjects = make(map[ObjectID]Object) // new cache.
-	ds.opslog = nil                             // new ops log.
+	ds.objectSizeCache = make(map[ObjectID]int64)
```

Comment:
Let's put this in ObjectInfo as unexposed `lastObjectSize` so as to avoid a map lookup.